src/main/java/org/folio/inventory/consortium/util/ConsortiumUtil.java

```diff
@@ -31,4 +39,14 @@ public static Future<Optional<SharingInstance>> createShadowInstanceIfNeeded(Con
         return Future.failedFuture(throwable);
       });
   }
+
+  public static CompletableFuture<OkapiHttpClient> createOkapiHttpClient(Context context, HttpClient httpClient) {
```

Comment:
could this use vertx future instead of java.util.concurrent.CompletableFuture?

Reply:
OkapiHttpClient uses CompletableFuture under the hood, so using vert.x Future could cause some difficulties composing Future and CompletableFuture

Reply:
I see. The difficulties are just moved elsewhere since mod-inventory ought to be a vert.x app.